src/derivative_operators/BC_operators.jl

```diff
@@ -139,14 +145,150 @@ function LinearAlgebra.Array(Q::BoundaryPaddedVector)
     return [Q.l; Q.u; Q.r]
 end
 
-function Base.convert(::Type{Array},A::AbstractBC{T}) where T
+function Base.convert(::Type{Array},A::SingleLayerBC{T}) where T
     Array(A)
 end
 
-function Base.convert(::Type{SparseMatrixCSC},A::AbstractBC{T}) where T
+function Base.convert(::Type{SparseMatrixCSC},A::SingleLayerBC{T}) where T
     SparseMatrixCSC(A)
 end
 
-function Base.convert(::Type{AbstractMatrix},A::AbstractBC{T}) where T
+function Base.convert(::Type{AbstractMatrix},A::SingleLayerBC{T}) where T
     SparseMatrixCSC(A)
 end
+
+#######################################################################
+# Multidimensional
+#######################################################################
+
+# One of my utility functions, it works by applying an operator along a slice of u in the dimension specified by n
+# A multidim convolution would be better. - this is unused in DifEqOperators.jl but used as a template for the following functions
```

Comment:
indeed, multidimensional convolutions (using NNLib.jl) is where we are going next on the GhostDerivativeOperators for `L1 + L2` combinations. But it shouldn't be necessary on the boundary.